app/src/main/java/com/onboarding/parkingsystemkotlin/fragment/ConfigureParkingLotsDialogFragment.kt

```diff
@@ -0,0 +1,43 @@
+package com.onboarding.parkingsystemkotlin.fragment
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.onboarding.parkingsystemkotlin.databinding.ConfigureParkingLotsDialogFragmentMainActivityBinding
+import com.onboarding.parkingsystemkotlin.listener.OnInputListener
+import com.onboarding.parkingsystemkotlin.mvp.contract.ConfigureParkingLotsContract
+import com.onboarding.parkingsystemkotlin.mvp.presenter.ConfigureParkingPresenter
+import com.onboarding.parkingsystemkotlin.mvp.view.ConfigureParkingView
+import com.onboarding.parkingsystemkotlin.utils.ConstantUtils
+
+class ConfigureParkingLotsDialogFragment : DialogFragment() {
+    private lateinit var presenter: ConfigureParkingLotsContract.ConfigureParkingPresenter
+    private lateinit var binding: ConfigureParkingLotsDialogFragmentMainActivityBinding
+    private lateinit var inputListener: OnInputListener
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        binding = ConfigureParkingLotsDialogFragmentMainActivityBinding.inflate(layoutInflater)
+        val view: View = binding.root
+        this.presenter = ConfigureParkingPresenter(inputListener = this.inputListener, view = ConfigureParkingView(this, this.binding))
+        setListeners()
+        return view
```

Comment:
you don't need view var, you can return binding.root

Reply:
removed